cdisutils/storage.py

```diff
@@ -8,23 +8,130 @@
 TODO: add tests
```

Comment:
Do we still have outstanding tests to write?

Reply:
That's possible but but I'm not sure. This comment was added 6 years ago by someone who no longer works here. The existing tests were changed to work with the `S3ConnectionProxyFix` class.